pycontw/pycontw-frontend
# feat/index-sponsor-cards

## Types of changes

* **New feature**




## Description


### Style 
* Update the layout of sponsor cards in index page
* Update style of heading 2
* Update style of SponsorModal

### Behavior
* Allow closing modal by clicking outside of the lightbox.

## Steps to Test This Pull Request
1. Go to the index page 
2. Scroll down to the Sponsors section
3. Click on cards and check the modal layout.

## Expected behavior


![localhost_3000_2021_en-us (2)](https://user-images.githubusercontent.com/18432820/164989668-1c7a43a3-4ee1-4a54-801a-51ed872c2e7f.png)


<img src="https://user-images.githubusercontent.com/18432820/164989562-ef17247c-ff8a-4157-b109-f5d82508d282.png" width=300.>


![image](https://user-images.githubusercontent.com/18432820/165063166-3b879b76-73d8-402e-a460-71dd69876b6b.png)

<img src="https://user-images.githubusercontent.com/18432820/165063219-a7cefae3-ea62-42ea-8876-c47258cbee65.png" width=300.>

## Related Issue


## Additional context
